src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd

```diff
@@ -1700,6 +1705,11 @@ elementFormDefault="qualified">
         </xs:annotation>
     </xs:element>
     <xs:element name="DocumentationFile" type="msb:StringPropertyType" substitutionGroup="msb:Property"/>
+    <xs:element name="EmitCompilerGeneratedFiles" type="msb:boolean" substitutionGroup="msb:Property">
+        <xs:annotation>
+            <xs:documentation><!-- _locID_text="EmitCompilerGeneratedFiles" _locComment="" -->Controls if source generated files will be saved.</xs:documentation>
```

Comment:
It looks like most of the other properties use "whether" instead of "if", so let's take that for consistency.

```suggestion
            <xs:documentation><!-- _locID_text="EmitCompilerGeneratedFiles" _locComment="" -->Controls whether source generated files will be saved.</xs:documentation>
```

Reply:
I love the consistency of our consistency feedback 😄